sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -159,8 +159,8 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
    * @param columnName the name of a column of integral type that will be used for partitioning.
    * @param lowerBound the minimum value of `columnName` used to decide partition stride
    * @param upperBound the maximum value of `columnName` used to decide partition stride
-   * @param numPartitions the number of partitions.  the range `minValue`-`maxValue` will be split
-   *                      evenly into this many partitions
+   * @param numPartitions the number of partitions, the range `lowerBound`-`upperBound` will be
```

Comment:
I updated the wording since numPartitions/lowerBound/upperBound aren't directly used in where clauses. It's a bit hard to be concise on the actual calculation in JDBCRelation, so I hope this is sufficient.